compiler/AST/expr.cpp

```diff
@@ -4468,6 +4469,8 @@ GenRet CallExpr::codegen() {
     case PRIM_ARRAY_GET:
     case PRIM_ARRAY_GET_VALUE:
     case PRIM_ON_LOCALE_NUM:
+    case PRIM_GET_REAL:
+    case PRIM_GET_IMAG:
       // generated during generation of PRIM_MOVE
```

Comment:
If the addition of the PRIM_GET_REAL/IMAG cases related to this PR, I am curious how.
Otherwise, it deserves a "while there" mention in the PR message.